integration/cli-tests/pom.xml

```diff
@@ -120,6 +120,54 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>2.6</version>
```

Comment:
Shall we not hard code the version here and inherit from parent pom

Reply:
Done.